app/src/main/java/com/koreatech/kwanhee_jo_compose_study/view/login/LoginPage.kt

```diff
@@ -103,6 +108,15 @@ fun LoginPage(
             onClick = onClickSignUpPage
         )
     }
+    when (state.status) {
+        UiStatus.Loading -> Unit
+        UiStatus.Success -> Unit
+        is UiStatus.Failed ->{
```

Comment:
![image](https://github.com/Pohanghang-Compose/kwanhee-jo-compose-study/assets/98825364/0fa207d7-9f14-445e-94f7-c27ec6994ac6)

commit check 🙏 

```suggestion
        is UiStatus.Failed -> {
```